XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -176,24 +192,84 @@ public override void OnPostRaycast()
 
             // Used to ensure the line doesn't extend beyond the cursor
             float cursorOffsetWorldLength = BaseCursor?.SurfaceCursorDistance ?? 0f;
+            
+            if (IsSelectPressed)
+            {
+                BeamAnimator.SetBool("triggerPull", true);
+
+                if(holdCounter<1f)
+                {
+                    #if PLATFORM_LUMIN     
+                    MLInput.GetController(0).StartFeedbackPatternVibe(MLInputControllerFeedbackPatternVibe.Click , MLInputControllerFeedbackIntensity.Low);
+                    #endif
+                    holdCounter++;
+                }
+
+                else
+                {
+                    MLInput.GetController(0).StopFeedbackPatternVibe();    
+                }
+            }
+
+            else 
+            {
+                BeamAnimator.SetBool("triggerPull", false);
+                holdCounter = 0;    
+            }
 
             // If we hit something
             if (Result.CurrentPointerTarget != null)
             {
                 clearWorldLength = Result.RayDistance;
-                lineColor = IsSelectPressed ? LineColorSelected : LineColorValid;
+                
+                //If object is marked as interactable (ie, a button that's not marked as inactive)
+                if(Result.CurrentPointerTarget.gameObject.tag == "Interactable")
```

Comment:
We shouldn't be using tags as a way to mark an object interactable. Better to use layers, but even better is that the object implements an input or focus handler that responds to the event.

Reply:
Tags are a little cleaner than layers imo, since there's a different use case w/ layers for collision, etc. Tags allow for updating/tagging/un-tagging a layer, without changing collision or rendering. 

My thinking was:
With a mechanic like apple's dock, you might want to overdraw and separate the physics for everything that's a child of the dock object- but sub-manage the icons within the dock, to flag as interactive/non-interactive based on states or conditions.

Reply:
The whole input system has a dedicated way to determine which objects get specific actions.
Let's see if we can update this to utilize that feature.